taskchampion/taskchampion/src/errors.rs

```diff
@@ -1,9 +1,13 @@
+use std::io;
 use thiserror::Error;
 
-#[derive(Debug, Error, Eq, PartialEq, Clone)]
```

Comment:
Is it ok that these traits were dropped? I removed them as a quick fix but it also wasn't clear to me what the benefit was. Or is it just best practice to include all possible derivable traits so you're conscientious of losing them?

Reply:
There are best practices for deriving traits when they are practical on public types.  Especially Debug is useful for consumers of a type.

Reply:
..but if the derivation isn't automatic anymore, which I guess was the case here, then it's fine to omit them.